docs/core/entity/cover.md

```
@@ -60,6 +61,7 @@ and are combined using the bitwise or (`|`) operator.
 | `OPEN`              | The cover supports being opened.                                                 |
 | `CLOSE`             | The cover supports being closed.                                                 |
 | `SET_POSITION`      | The cover supports moving to a specific position between opened and closed.      |
+| `SPEED`             | The cover supports passing a speed parameter to open, close, set_position actions     |
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_only; Actual: leading_and_trailing; Unexpected trailing pipe

(MD055, table-pipe-style)